### ISSUE TITLE
vcard 2.1 support: add

### ISSUE THREAD

**Issue by icegood** [eventable/vobject#168]
Some (older) phones are export images in vcards in 2.1 format with line
ENCODING=BASE64

instead of

ENCODING=B
(for v 3.0: https://datatracker.ietf.org/doc/html/rfc2426#section-3.1.4)
in PHOTO field.

**Comment by slyduda** [comment]
The following quick fix helped me:

https://github.com/Kozea/Radicale/issues/1238#issuecomment-1352530048

**Comment by pbiering** [comment]
> The following quick fix helped me:
> 
> [Kozea/Radicale#1238 (comment)](https://github.com/Kozea/Radicale/issues/1238#issuecomment-1352530048)

> The encoding gets all wonky when BASE64 is used for the encoding param. I am not sure why and I am sure this quick fix breaks other cases, but it allows me to use base64 encoding on photos now so that's all that matters for me.

=> one has to investigate proper solution for "vobject" - unconditional overtake of the "quick fix" can turn dangerous.

**Comment by slyduda** [comment]
agreed, and thank you for the clarification. i should have added a bit more words of caution on my comment!
